bitcoin/src/p2p/message.rs

```diff
@@ -310,9 +310,9 @@ impl RawNetworkMessage {
         Self { magic, payload, payload_len, checksum }
     }
 
-    /// Consumes the [RawNetworkMessage] instance and returns its inner components.
-    pub fn deconstruct(self) -> (Magic, NetworkMessage, u32, [u8; 4]) {
-        (self.magic, self.payload, self.payload_len, self.checksum)
+    /// Consumes the [RawNetworkMessage] instance and returns the inner payload.
```

Comment:
```suggestion
    /// Consumes the [`RawNetworkMessage`] instance and returns the inner payload.
```

Now it's doc-linked

Reply:
I was trying to keep the styling consistent within this file. It may be _fixed_ in another PR, along with the other same usages.